packages/stryker-mocha-runner/src/MochaOptionsLoader.ts

```diff
@@ -7,18 +7,28 @@ import MochaRunnerOptions, { mochaOptionsKey } from './MochaRunnerOptions';
 export default class MochaOptionsLoader {
 
   private readonly log = getLogger(MochaOptionsLoader.name);
+  private readonly DEFAULT_MOCHA_OPTS = 'test/mocha.opts';
 
   public load(config: StrykerOptions): MochaRunnerOptions {
     const mochaOptions = Object.assign({}, config[mochaOptionsKey]) as MochaRunnerOptions;
+    let optsFileName = path.resolve(this.DEFAULT_MOCHA_OPTS);
+
     if (mochaOptions && mochaOptions.opts) {
-      const optsFileName = path.resolve(mochaOptions.opts);
+      optsFileName = path.resolve(mochaOptions.opts);
+    }
+
+    if (fs.existsSync(optsFileName)) {
       this.log.info(`Loading mochaOpts from "${optsFileName}"`);
       const options = fs.readFileSync(optsFileName, 'utf8');
       return Object.assign(this.parseOptsFile(options), mochaOptions);
+    }
+
+    if (mochaOptions && mochaOptions.opts) {
+      throw new Error(`Could not load opts from "${optsFileName}". Please make sure opts file exists.`);
```

Comment:
This looks really strange to me. Because we return on line 23, we throw an error here.

If we get an error while reading or parsing the `Opts` file, we will not log this error because we have no try-catch. 

Could you change this to:
```
if (fs.existsSync(optsFileName)) {
   ...
} else if (mochaOptions.opts) {
   this.log.error(`Could not load opts from "${optsFileName}". Please make sure opts file exists.`);
}
```
(The stryker-karma-runner handles an error like this by logging it instead of throwing an error)